tests/unit/cycling/test_iso8601.py

```diff
@@ -855,63 +829,37 @@ def test_weeks_days(set_cycling_type):
     ]
 
 
-def test_cug(set_cycling_type):
+@pytest.mark.parametrize(
+    'value, expected', [
+        ('next(T-00)', '20180314T1600Z'),
+        ('previous(T-00)', '20180314T1500Z'),
+        ('next(T-00; T-15; T-30; T-45)', '20180314T1515Z'),
+        ('previous(T-00; T-15; T-30; T-45)', '20180314T1500Z'),
+        ('next(T00)', '20180315T0000Z'),
+        ('previous(T00)', '20180314T0000Z'),
+        ('next(T06:30Z)', '20180315T0630Z'),
+        ('previous(T06:30) -P1D', '20180313T0630Z'),
+        ('next(T00; T06; T12; T18)', '20180314T1800Z'),
+        ('previous(T00; T06; T12; T18)', '20180314T1200Z'),
+        ('next(T00; T06; T12; T18)+P1W', '20180321T1800Z'),
+        ('PT1H', '20180314T1612Z'),
+        ('-P1M', '20180214T1512Z'),
+        ('next(-00)', '21000101T0000Z'),
+        ('previous(--01)', '20180101T0000Z'),
+        ('next(---01)', '20180401T0000Z'),
+        ('previous(--1225)', '20171225T0000Z'),
+        ('next(-2006)', '20200601T0000Z'),
+        ('previous(-W101)', '20180305T0000Z'),
+        ('next(-W-1; -W-3; -W-5)', '20180314T0000Z'),
+        ('next(-001; -091; -181; -271)', '20180401T0000Z'),
+        ('previous(-365T12Z)', '20171231T1200Z'),
+    ]
+)
+def test_cug(value: str, expected: str, set_cycling_type):
```

Comment:
I don't like the acronym.

```suggestion
def test_user_guide_examples(value: str, expected: str, set_cycling_type):
```

Might be a good idea to put a URL to the page of the User Guide in question in the docstring?